base/freestanding/femr/config/fhir/application.yaml

```diff
@@ -90,34 +134,39 @@ hapi:
     search-coord-max-pool-size: 100
     search-coord-queue-capacity: 200
 
-#    logger:
-#      error_format: 'ERROR - ${requestVerb} ${requestUrl}'
-#      format: >-
-#        Path[${servletPath}] Source[${requestHeader.x-forwarded-for}]
-#        Operation[${operationType} ${operationName} ${idOrResourceName}]
-#        UA[${requestHeader.user-agent}] Params[${requestParameters}]
-#        ResponseEncoding[${responseEncodingNoDefault}]
-#      log_exceptions: true
-#      name: fhirtest.access
-#    max_binary_size: 104857600
-    max_page_size: 1000
```

Comment:
another potential unintended change (`max_page_size`)?  as we're more competent with paging, probably don't need this any longer anyhow...

Reply:
Nice catch! I'm hopeful we don't need this anymore :crossed_fingers: , but I'll keep it in mind if we see any issues with paging